Source/RapyutaSimulationPlugins/Public/Tools/LimitRTFFixedSizeCustomTimeStep.h

```diff
@@ -0,0 +1,112 @@
+/**
+ * @file LimitRTFFixedSizeCustomTimeStep.h
+ * @brief CustomTimeStep class which uses fixed time step and and limit RTF(Real Time Factor)
+ * @copyright Copyright 2020-2022 Rapyuta Robotics Co., Ltd.
+ */
+
+#pragma once
+#include "Engine/EngineCustomTimeStep.h"
+
+#include "LimitRTFFixedSizeCustomTimeStep.generated.h"
+
+class UEngine;
+
+/**
+ * @brief Control the Engine TimeStep via a fixed time step and limit RTF(Real Time Factor).
+ * Main logic is copied from UGenlockedFixedRateCustomTimeStep and UEngineCustomTimeStep.
+ * @sa [UEngineCustomTimeStep](https://docs.unrealengine.com/4.27/en-US/API/Runtime/Engine/Engine/UEngineCustomTimeStep/)
+ * @sa [UGenlockedFixedRateCustomTimeStep](https://docs.unrealengine.com/4.27/en-US/API/Runtime/TimeManagement/UGenlockedFixedRateCustomTimeSte-/)
+ * 
+ */
+UCLASS(Blueprintable, editinlinenew, meta = (DisplayName = "Limit RTF Fixed Rate"))
+class TIMEMANAGEMENT_API ULimitRTFFixedSizeCustomTimeStep : public UEngineCustomTimeStep

```

Comment:
```suggestion
class RAPYUTASIMULATIONPLUGINS_API URRLimitRTFFixedSizeCustomTimeStep : public UEngineCustomTimeStep
```
`RR` prefix is important 🙏🏼